Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -214,6 +214,18 @@ public void DoGetRequest()
     ///   </code>
     ///   <para>Copies the contents of the source as long as the source and destination schemas match.</para>
     /// </example>
+    /// <example>
+    ///   <code>
+    /// PS C:\> $job = $table | Start-BqJob -Load CSV "$gcspath/basic.csv" -SkipLeadingRows 1 -Synchronous
```

Comment:
I think it's a little confusing having variables that aren't defined in the example. Can they be defined?